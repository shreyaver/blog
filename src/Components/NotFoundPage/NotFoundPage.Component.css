.Not-found-page{
    width: 70%;
    min-width: 300px;
    height: 47vw;
    min-height: 830px;
    margin: auto;
    display: flex;
    flex-direction: column;
    flex-wrap: wrap;
    justify-content: center;
    align-items: center;
    color: rgb(115, 115, 115);
    text-transform: uppercase;
    font-size: 18px;
}