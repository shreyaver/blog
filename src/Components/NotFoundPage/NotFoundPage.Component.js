import React from 'react';
import Header from '../Header/Header.Component';
import Footer from '../Footer/Footer.Component';
import './NotFoundPage.Component.css';

const NotFoundPage = (props) => {
  return (
    <div>
      <Header />
      <div className = "Not-found-page"> 
      {/* <i class="icomoon icon-confused"></i> */}
      Page Not Found
      </div>
      <Footer />
    </div>
  );
}

export default NotFoundPage;