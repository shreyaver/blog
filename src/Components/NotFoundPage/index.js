import React from 'react';
import Header from '../Header';
import Footer from '../Footer';
import './NotFoundPage.css';

const NotFoundPage = (props) => {
  return (
    <div>
      <Header />
      <div className = "Not-found-page"> 
      {/* <i class="icomoon icon-confused"></i> */}
      Page Not Found
      </div>
      <Footer />
    </div>
  );
}

export default NotFoundPage;