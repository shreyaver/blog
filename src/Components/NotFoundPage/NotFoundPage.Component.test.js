import React from 'react';
import renderer from 'react-test-renderer';
import NotFoundPage from './NotFoundPage.Component.js';

describe('NotFoundPage', () => {
  it('renders without crashing', () => {
    const notFoundPage = renderer.create(<NotFoundPage />).toJSON();
    expect(notFoundPage).toMatchSnapshot();
  });
});
