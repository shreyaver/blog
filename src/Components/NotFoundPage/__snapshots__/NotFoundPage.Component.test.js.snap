// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage renders without crashing 1`] = `
<div>
  <header>
    <div
      className="Content"
    >
      <div>
        <h1>
          The Artifact
        </h1>
        <h2>
          Culture & Art blog
        </h2>
      </div>
      <nav>
        <div
          className="navBar"
        >
          <a
            href="/"
          >
            Blog
          </a>
          <a
            href="#"
          >
            About
          </a>
          <a
            href="#"
          >
            Contact
          </a>
        </div>
      </nav>
    </div>
  </header>
  <div
    className="Not-found-page"
  >
    Page Not Found
  </div>
  <footer>
    <div
      className="footerContent"
    >
      <small>
        Â©artifact.com 2019
      </small>
    </div>
  </footer>
</div>
`;
